deephaven_ipywidgets/deephaven.py

```diff
@@ -138,11 +138,15 @@ def __init__(self, deephaven_object, height=600, width=0, session=None):
             server_url, token = _check_session(session, params)
 
         elif _str_object_type(deephaven_object) == "pydeephaven.table.Table":
+            from pydeephaven.session import SharedTicket
             session = deephaven_object.session
 
             server_url, token = _check_session(session, params)
 
-            session.bind_table(object_id, deephaven_object)
+            ticket = SharedTicket(b"h/" + object_id.encode("utf-8"))
```

Comment:
I'm not entirely sure myself, I saw you made this comment https://github.com/deephaven/deephaven-core/pull/5854#discussion_r1701895046 so I included it

Reply:
And this doesn't work for other tickets at this time. I think that requires some other mechanism that we either don't have or is tricky to do. I tried using `publish` (see implementation of (publish_table)[https://deephaven.io/core/client-api/python/code/pydeephaven.session.html#pydeephaven.session.Session.publish_table]) but it requires a ticket to already be attached to the object. I didn't really investigate beyond that because it seems like something that requires more careful thought.

Reply:
added comment